blog/2025/2025-05-09-blender-baking/index.md

```diff
@@ -0,0 +1,222 @@
+
+For this tutorial we are going to use this wooden bowl model from the fantastic website [Polyhaven](https://polyhaven.com/a/wooden_bowl_02)
+
+![Bowl Rendered](bowl-rendered.webp)
+
+<!-- truncate -->
+
+As with all of the free CC0 models on Polyhaven this mesh has a fairly high number of small triangles, 4,666 to be exact, while that may not seem like a lot think about how big it is likely to be in a realtime scene. Most of the time the entire bowl might only be a few pixels tall! Especially given that small triangles are much more expensive than large triangles (due to quad occupancy) this is probably something we should deal with.
+
+![Small Bowl](small-bowl.webp)
+
+Now that we understand why we must bake, lets go ahead and do it.
+
+# Preparing for the Bake
+
+Open up your high poly model in Blender, I am using Blender 4.4, other versions should work but your UI might not match up exactly with this tutorial.
+
+![High Poly Blender](high-poly-blender.webp)
+
+You then will need a low poly version of the model. How create a low poly model is outside the scope of this tutorial, but it *must* be UV-unwrapped before proceeding and none of the polygons should be overlapping on the UV map.
+
+![Low Poly Blender](low-poly-blender.webp)
+
+My low-poly version uses only 272 triangles or roughly 5.8% of the original number. These are also bigger triangles so should have much better quad occupancy on the GPU.
+
+![overlapping models](overlapping-models.webp)
+
+The first thing you need to do is make sure the high and low-poly models are directly on top of each other, just like in the image above. You must also ensure the scale of the low poly version is exactly 1.0 on all axes.
+
+![Unit Scale](unit-scale.webp)
+
+If it is not you can apply the scale with Ctrl+A -> Apply -> Scale, while the low poly object is selected in object mode. 
+
+![triangulate](triangulate.webp)
+
+Next you need to add a triangulate modifier to the low poly object, the exact options you pick here don't really matter but if you change them after the bake you must re-bake all maps.
+
+# Creating a Bake Target Proxy
+
+The main way my workflow differs from what I've seen elsewhere is the use of a Bake Target or Proxy, this is not strictly necessary but it makes the entire process far less frustrating if you need to run the baking process more than once, which you inevitably will. This involves creating a linked duplicate to our low poly object, this will allow you to preview the bake results without having to mess around with the shader nodes and having to reconnect things between bakes.
+
+![Linked Duplicate](linked-duplicate.webp)
+
+To create a linked duplicate, simply select your low-poly object and hit Alt+D, you can then move your linked duplicate off to the side somewhere.
+
+![Outliner Names](outliner-names.webp)
+
+I'm going to name the new object "Low Poly" and the first one "Bake Target" (The names don't matter but it's nice to be organized)
```

Comment:
I wonder if we should put all the names you set everywhere like "Low Poly" and "Bake Target" inside ```code``` tags to make them more visible